libs/extensions-registry/models.ts

```diff
@@ -171,6 +171,6 @@ export interface ManifestWithMeta extends ManifestBase {
 }
 
 export interface ManifestEntrypoint extends ManifestBase {
-	type: 'entrypoint';
+	type: 'entryPoint';
```

Comment:
Please, all our test packages will stop working, do we have to rename this? 😂

Reply:
Test packages? Where do we have those?

Reply:
Sorry, I meant packages on various installations. For instance, I have a v13 branch on my machine with some packages and I distributed those as demo files for the Spark presentation.

Do you feel that this change clarifies what this kind of extension is?

Reply:
Ah, I didn't know that. I just noticed that I misspelled this a couple of times so I just wanted to get this updated before we had too many extensions developed.

Reply:
In general, it seems that "entry point" as two words is most correct. Though it seems one word is acceptable. But I don't think we should respect previous matters at this stage. So I'm in for the most correct at the time being. With the perspective that we are still free to adjust anything, then we should keep that freedom and don't spend time evaluating changes jet. IMO.